src/providers/completer/input.ts

```diff
@@ -40,12 +45,12 @@ export class Input {
 
     getGraphicsPath(filePath: string) {
         const content = utils.stripComments(fs.readFileSync(filePath, 'utf-8'), '%')
-        const regex = /\\graphicspath{(.*)}/g
-        let result: string[]|null
+        const regex = /\\graphicspath{(([^\{\}]|{.*})*)}/g
+        let result: string[] | null
         do {
             result = regex.exec(content)
             if (result) {
-                for (const dir of result[1].split(/\{|\}/).filter(s => s)) {
+                for (const dir of result[1].split(/\{|\}/).filter(s => s.replace(/^\s*$/, ''))) {
```

Comment:
What is the purpose of this change?

Reply:
For a multiline graphicspath,
```tex
\graphicspath{
    {somepath},
    {anotherpath}
}
```
matches which entirely consist of whitespace will be produced. This makes sure they are ignored.